image/pubspec.yaml

```diff
@@ -6,14 +6,14 @@ _author: David PHAM-VAN <dev.nfet.net@gmail.com>
 homepage: https://github.com/DavBfr/dart_barcode/tree/master/image
 repository: https://github.com/DavBfr/dart_barcode
 issue_tracker: https://github.com/DavBfr/dart_barcode/issues
-version: 2.0.3
+version: 2.0.4
 
 environment:
   sdk: ">=2.12.0 <4.0.0"
 
 dependencies:
   args: ^2.3.0
-  barcode: ^2.2.4
+  barcode: ^2.2.5
```

Comment:
You don't need to change this file. Pub will take the latest version

Reply:
I have removed this part of the edit in [b286841](https://github.com/DavBfr/dart_barcode/pull/65/commits/b2868410d5f09fdc5ec53684aa63439cdd4d29b7) .